accounts/hw/src/trezor.rs

```diff
@@ -118,6 +118,12 @@ enum HidVersion {
 	V2,
 }
 
+fn encode_message<M: Message>(message: M) -> Vec<u8> {
+	let mut result = Vec::with_capacity(message.encoded_len());
+	message.encode(&mut result).expect("Vec has unlimited capacity; qed");
```

Comment:
Nitpick, I don't agree with `unlimited capacity` because `Vec::alloc` will panic when capacity is bigger than `isize::max_value` on 32-bit platforms and `usize::max_value` on 64-bit platforms.

However, I think that is `only` theoretically and would be fine with some like `message could be represented as Vec there encodable; qed`